client/app/lib/value-format.js

```diff
@@ -2,6 +2,8 @@ import moment from 'moment/moment';
 import numeral from 'numeral';
 import _ from 'underscore';
 
+numeral.options.scalePercentBy100 = false;
```

Comment:
What's this?

Reply:
By default, for formats like `0[.]00%` `numeral` will multiply value by 100 - so `0.5` will become `50%`. This option disables this behavior so `0.5` will become `0.5%`, and `50` - `50%`.